hooks/usePinata.js

```diff
@@ -0,0 +1,70 @@
+// custom hook for IPFS
+import { useEffect, useState } from "react";
+import { pinFileToIPFS, unpinPinata } from "../lib/ipfsPinata";
+import { upload } from "../lib/crust";
+
+
+// Change this for using crust or not. If you use crust we will unpin the file from pinata
+const deployCrust = false;
```

Comment:
Setting deployCrust to false here and setting it to true at `pinFileToIPFS` would result to endless call of the hook